scripts/post-update.sh

```diff
@@ -277,7 +277,6 @@ if (( $PRE_0_0_16 > 0 )); then
     # append 'enable_uart=1' line to /boot/config.txt
     echo "enable_uart=1" | sudo tee -a /boot/config.txt
 
-    source ~/.bashrc
```

Comment:
I think this is importation for us while developing.

Reply:
This only applies to the current shell, which is detached from any user interaction right here, and then exits. This shell doesn't need to use the alias. Any shell that is opened after the update will already have sourced it automatically.

Reply:
The aliases are still available in any shell after the update, I tested it.